app/server/routes/core.ts

```diff
@@ -21,4 +22,23 @@ router.get("/_prout", (_, res: Response) => {
   res.send(`<pre>${GIT_COMMIT_HASH}</pre>`);
 });
 
+router.get("/robots.txt", (_, res: Response) => {
+  const disallowAll = "UserAgent: *\n" + "Disallow: /\n\n";
+  const allowHelpCentre =
+    "UserAgent: *\n" +
+    "Allow: /help-centre\n" +
+    "Allow: /help-centre/\n" +
+    "Disallow: /\n\n";
+  const disallowGoogleAdsBots =
+    "User-agent: AdsBot-Google\n" +
+    "User-agent: AdsBot-Google-Mobile\n" +
+    "Disallow: /\n\n";
+  const prodAccessList = allowHelpCentre + disallowGoogleAdsBots;
+  const preProdAccessList = disallowAll;
```

Comment:
I believe `nonProdAccessList` would be clearer name for this. Not all code goes through other stages before going to PROD.

Reply:
I was thinking pre-prod as opposed to prod or post-prod, so any stage before prod regardless of whether there actually is such a stage.